_infra/helm/collection-exercise/values.yaml

```diff
@@ -72,4 +72,7 @@ gcp:
   collectionExerciseEndTopic: collection-exercise-end
   collectionExerciseEventStatusUpdateSubscription: collection-exercise-event-update
   supplementaryDataServiceTopic: supplementary-data-service-topic
-  supplementaryDataServiceSubscription: supplementary-data-service-subscription
\ No newline at end of file
+  supplementaryDataServiceSubscription: supplementary-data-service-subscription
+
+loadBalancer:
+  ipAddress: "10.110.128.12"
```

Comment:
Asd this is very specific internal config per env we shouldn't default this to anything here. It should only ever be provided by config.

Reply:
This IP is specifically only for dev and sandbox and will be replaced with a placeholder string once we upgrade to GKE 1.29

Reply:
Still not convinced this key is needed here at all.